bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -303,7 +292,11 @@ impl Into<*const ffi::evmc_result> for ExecutionResult {
             output_data: buffer,
             output_size: len,
             release: Some(release_heap_result),
-            create_address: self.create_address,
+            create_address: if self.create_address.is_some() {
```

Comment:
Just a cosmetic comment: I think this can be done more cleanly with an `if let` pattern.

Reply:
How is the `if let` pattern any clearer?

Reply:
personal opinion.